sonar-project.properties

```
@@ -2,3 +2,4 @@ sonar.projectKey=kkdz
 sonar.organization=kkdz
 
 sonar.javascript.lcov.reportPaths=./coverage/lcov.info
+sonar.coverage.exclusions=./cypress
```

Looking at [the CI logs](https://github.com/se310-Team4/KKDZ/pull/56/checks#step:7:59) for this PR, it says `INFO: 0 files ignored because of inclusion/exclusion patterns`.  So I don't think this fix has worked. Based on [this article](https://community.sonarsource.com/t/sonar-coverage-exclusions-not-excluding-too-much/31276/8), I think the syntax needs to be:

```suggestion
sonar.coverage.exclusions=**/cypress/**/*.*
```